code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -5,7 +5,7 @@
 	id = "ipc"
 	say_mod = "states" //inherited from a user's real species
 	sexes = FALSE
-	species_traits = list(NOTRANSSTING,NOEYESPRITES,NO_DNA_COPY,NOZOMBIE,MUTCOLORS,NOHUSK,AGENDER,NOBLOOD,NO_UNDERWEAR)
```

Comment:
Removing NOBLOOD changes a number of things

- prevents rolling bloodsucker and vampire
- allows bloodloss to kill them
- gives them a heart and makes them require a heart to live
- blood cult

It changes far too many things about the species
if you want a robot species to have weird blood, use preternis

Reply:
fixed it